File: examples/python/django/app/ride_share/views/car/car.py
from rest_framework.views import APIView
from rest_framework.response import Response

from ride_share.utility.utility import find_nearest_vehicle

class CarView(APIView):
    """
    View to order a car
    """

    def order_car(self, search_radius):
        find_nearest_vehicle(search_radius, "car")

    def get(self, request, format=None):
        """
        Order a car within a certain radius.
        """
        self.order_car(0.4)
        return Response(["Car view"])

File: examples/language-sdk-instrumentation/golang-push/rideshare/car/car.go
package car

import (
	"context"
	"rideshare/rideshare"
	"rideshare/utility"
)

func OrderCar(ctx context.Context, searchRadius int64) {
	rideshare.Log.Printf(ctx, "ordering car, with searchRadius=%d", searchRadius)
	utility.FindNearestVehicle(ctx, searchRadius, "car")
}
